playbooks/openshift/cinder-registry.yml

```diff
@@ -1,20 +1,28 @@
 --- 
 # Create a volume for the registry
 - hosts: localhost
-  roles: 
+  vars:
+    osp_volumes:
+    - name: "{{ env_id }}.{{ dns_domain }}-registry"
```

Comment:
consider using `full_dns_domain` here (and below) - as seen [here](https://github.com/redhat-cop/casl-ansible/blob/master/playbooks/openshift/pre-tasks.yml#L25)